Readme.md

```diff
@@ -258,7 +258,8 @@ Options are:
         --failure-exit-code INT      Specify the exit code to use when tests fail
         --specify-groups SPECS       Use 'specify-groups' if you want to specify multiple specs running in multiple
                                      processes in a specific formation. Commas indicate specs in the same process,
-                                     pipes indicate specs in a new process. Cannot use with --single, --isolate, or
+                                     pipes indicate specs in a new process. If SPECS is a `-` the actual value for this
```

Comment:
```suggestion
                                     pipes indicate specs in a new process. If SPECS is a `-` the value for this
```

Reply:
Also, does the CLI parser need to have its help updated?

Reply:
done. sorry for missing this. how silly.